src/web/aws/cloudwatch/kinesis/ExistingStreams.jsx

```diff
@@ -96,9 +96,9 @@ const KinesisStreams = ({ onChange, onSubmit, toggleSetup }) => {
                       Below is a list of all Kinesis streams found within the specified AWS account.
                     </p>
                     <p>
-                      Please select the stream you would like to read messages from, or follow the directions set up&nbsp;
-                      <a href={Routes.INTEGRATIONS.AWS.CLOUDWATCH.step('kinesis-setup')}>CloudWatch Log Subscription</a>,
-                      which can forward messages into a new Kinesis stream.
+                      Please select the stream you would like to read messages from, or follow the&nbsp;
+                      <a href="https://docs.graylog.org/en/latest/pages/integrations/inputs/aws_kinesis_cloudwatch_input.html"> AWS Kinesis/CloudWatch Input </a>
```

Comment:
@ceruleancee Typically, the links are configured in a json file. we need to look in other places, how routes are configured.

Reply:
Routes can be configured in `src/web/aws/common/Routes.js`

Reply:
@kyleknighted - Thanks for the pointer.
@ceruleancee Here's an stack overflow example on how to configure a external link, https://stackoverflow.com/questions/42914666/react-router-external-link